spec/controllers/api/v1/people_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe API::V1::PeopleController do before(:each) { request.headers['Accept'] = "application/vnd.birthday_finder_api.v1" }
```

Comment:
Line is too long. [124/80]<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.